tests/publiccloud/prepare_tools.pm

```diff
@@ -46,9 +46,13 @@ sub run {
         assert_script_run("pip3 install -q keyring");
     }
     elsif (is_sle) {
-        zypper_call('rr Cloud_Tools');
-        zypper_call('ref');
         zypper_call('-q in aws-cli');
+
+        # boo#1122199
+        zypper_call('ar http://download.opensuse.org/repositories/devel:/languages:/python:/aws/' . generate_version() . '/devel:languages:python:aws.repo');
+        zypper_call('--gpg-auto-import-keys -q in -f --repo devel_languages_python_aws python-s3transfer');
```

Comment:
mmm so you adding repo , refreshing it and remove it ? what I missing ?

Reply:
also check `zypper_ar` function for adding repos

Reply:
I will use `zypper_ar`.
The second one is a `zypper in`.

Reply:
ah yeah now I see , sorry I overlook it